subsys/bluetooth/controller/ll_sw/pdu.h

```diff
@@ -185,6 +185,8 @@
 #define EVENT_CLOCK_JITTER_US   2
 /* Event interframe timings */
 #define EVENT_IFS_US            150
+/* Event interframe timings, default */
+#define EVENT_IFS_DEFAULT_US    EVENT_IFS_US
 /* Standard allows 2 us timing uncertainty inside the event */
 #define EVENT_IFS_MAX_US        (EVENT_IFS_US + EVENT_CLOCK_JITTER_US)
```

Comment:
What's the purpose of the duplicated `#define`? 

If we intend to keep it, should that be used for `EVENT_IFS_MAX_US` since that refers to a "standard"? 

Will these 2 ever be different?

Reply:
Looks like there is inconsistent use of `EVENT_IFS_US`, `EVENT_IFS_MAX_US`, `+ 4, `- 4` and `EVENT_CLOCK_JITTER_US`  .... this needs clean, but out side the scope of this PR.